django/core/cache/backends/memcached.py

```diff
@@ -164,6 +166,11 @@ def validate_key(self, key):
 class MemcachedCache(BaseMemcachedCache):
     "An implementation of a cache binding using python-memcached"
     def __init__(self, server, params):
+        warnings.warn(
+            'MemcachedCache is deprecated in favor of PyLibMCCache and '
+            'PyMemcacheCache.',
```

Comment:
In the changes you've made to the documentation examples it looks like we're preferring `PyMemcacheCache` to `PyLibMC`. Perhaps we should mention it first of the two in all documentation and warnings? Then when we make the release it'll look more like we've switched out an unmaintained library for a maintained one.

```suggestion
            'MemcachedCache is deprecated in favor of PyMemcacheCache and '
            'PyLibMCCache.',
```

Reply:
I've just kept the alphabetical order, but we can change this.

Reply:
Yeah. I think it was coincidentally alphabetical and more likely the order that they were implemented in.